src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java

```diff
@@ -0,0 +1,131 @@
+        if(args.length == 0){
+            sendGuildInfo(guild.getGuild(), channel);
+        } else {
+            String guildid = args[0];
+            Guild targetGuild = FlareBot.getInstance().getGuildByID(guildid);
+            if(targetGuild != null){
+                sendGuildInfo(targetGuild, channel);
+            } else {
+                MessageUtils.sendErrorMessage("We couldn't find that guild.", channel);
+            }
+        }
+    }
+
+    private void sendGuildInfo(Guild guild, TextChannel channel){
+        EmbedBuilder eb = new EmbedBuilder();
+        eb.setTitle(guild.getName());
+        eb.setThumbnail(guild.getIconUrl());
+        eb.addField("Users", "**Total:** " +
+                guild.getMembers().size() + "\n" +
+                "\n" +
+                "**Online:** " +
+                guild.getMembers().stream().filter(member -> !member.getOnlineStatus().equals(OnlineStatus.OFFLINE)).count() + "\n" +
+                "\n" +
+                "**Owner:** " +
+                guild.getOwner().getUser().getName() +  "#" + guild.getOwner().getUser().getDiscriminator(), true);
+        String afk = guild.getAfkChannel() == null ? "" :
+                "**AFK:**\n" +
+                        "Channel: " +
+                        guild.getAfkChannel().getName() + "\n" +
+                        "\n" +
+                        "Timeout: " +
+                        FlareBot.getInstance().formatTime(guild.getAfkTimeout().getSeconds(), TimeUnit.SECONDS, true, false);
+        eb.addField("Channels", "**Text**\n" +
+                "Total: " +
+                guild.getRoles().size() + "\n" +
+                "\n" +
+                "Default: " +
+                "#" + guild.getTextChannelById(guild.getId()).getName() + "\n" +
+                "\n" +
+                "**Voice**\n" +
+                "Total: " +
+                guild.getVoiceChannels().size() + "\n" +
+                "\n" +
+                "Active: " +
+                guild.getVoiceChannels().stream().filter(voiceChannel -> voiceChannel.getMembers().size() > 0).count() + "\n" +
+                "\n" + afk, true);
+        eb.addField("Misc info", "**Creation time:** " +
+                guild.getCreationTime().toLocalDateTime().atOffset(ZoneOffset.UTC).format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")) + " GMT/BST\n" +
+                "\n" +
+                "**Roles:** " +
+                guild.getRoles().size() + "\n" +
+                "\n" +
+                "**Server region:** " +
+                guild.getRegion().getName() + "\n" +
+                "\n" +
+                "**Verification Level:** " +
+                getVerificationString(guild.getVerificationLevel()), true);
+        eb.setFooter("ID: " + guild.getId(), null);
+        eb.setColor(Color.BLUE);
+        channel.sendMessage(eb.build()).queue();
+    }
+
+    private String getVerificationString(Guild.VerificationLevel level){
+        switch (level){
```

Comment:
Before anyone else comments on this. JDA doesn't really have a better way to do it that i found